RadarSDK/RadarAPIClient.m

```diff
@@ -496,4 +497,39 @@ - (void)ipGeocodeWithCompletionHandler:(RadarIPGeocodeAPICompletionHandler)compl
     }];
 }
 
+- (void)getContextWithLocation:(CLLocation * _Nonnull)location
+             completionHandler:(RadarGetContextAPICompletionHandler _Nullable)completionHandler {
+    NSString *publishableKey = [RadarSettings publishableKey];
+    if (!publishableKey) {
+        return completionHandler(RadarStatusErrorPublishableKey, nil, nil);
+    }
+
+    NSMutableString *queryString = [NSMutableString new];
+    [queryString appendFormat:@"coordinates=%.15f,%.15f", location.coordinate.latitude, location.coordinate.longitude];
+
+    NSString *host = [RadarSettings host];
+    NSString *url = [NSString stringWithFormat:@"%@/v1/context?%@", host, queryString];
+    
+    NSDictionary *headers = @{
+        @"Authorization": publishableKey,
+        @"Content-Type": @"application/json",
+        @"User-Agent": [RadarUtils userAgent],
+        @"X-Radar-Config": @"true",
+    };
+    
+    [self.apiHelper requestWithMethod:@"GET" url:url headers:headers params:nil completionHandler:^(RadarStatus status, NSDictionary * _Nullable res) {
```

Comment:
wonder if our `apiHelper` is smart enough to construct the params differently between `GET` and other requests, doesn't have to be in this PR, but then we'd be able to move code such as https://github.com/radarlabs/radar-sdk-ios/blob/0e09f35c4cd35aec42603f150d6cf716770d277d/RadarSDK/RadarAPIClient.m#L507-L508 into that layer instead of here